parse.go

```diff
@@ -92,9 +97,15 @@ func addFuncDeclComment(fd *ast.FuncDecl, commentTemplate string) {
 
 }
 
-func addValueSpecComment(gd *ast.GenDecl, vs *ast.ValueSpec, commentTemplate string) {
+func addValueSpecComment(gd *ast.GenDecl, vs *ast.ValueSpec, commentTemplate string,
```

Comment:
I don't think we need `parentFlag` here, please keep the code consistent with comment template, instead of `define of ...`.

Reply:
ok